src/routes/protected.route.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import authMiddleware from '../middlewares/auth.middleware';
+
+const { isloggedIn } = authMiddleware;
+
+const app = express.Router();
+app.get('/protected', isloggedIn, (req, res) => {
```

Comment:
This can be moved to the auth route instead.

Reply:
ok